validate/validator.py

```diff
@@ -213,7 +213,15 @@ def string_template(option_value, name):
     try:
         uuid.UUID({{ name }})
     except ValueError:
-        raise ValidationFailed(\"{{ name }} is not a valid UUID\")     
+        raise ValidationFailed(\"{{ name }} is not a valid UUID\")
```

Comment:
Another idea that occurred to me; can't we have PGV translate from the well known pattern name to a regex in the protoc plugin prior to calling any of the language-specific code generators? That way we can leverage the existing regex code generation without much new code.